lib/content-services/social/like.component.scss

```diff
@@ -1,42 +1,41 @@
+@mixin selected {
+    color: #2196f3;
+}
+
+@mixin unselected {
+    color: #808080;
+}
+
```

Comment:
As an improvement,  instead of blocks with a single attribute, could we have them as https://github.com/Alfresco/alfresco-ng2-components/blob/7f4e301ec0574370f335d74b41ad986f843f3b95/lib/core/layout/components/sidenav-layout/sidenav-layout.component.scss#L2

This way it becomes more readable. It could be something like `$adf-rated-color:#2196f3 => .adf-like-select {color: $adf-rated-color}`